packages/coordinator/tests/services/translation/xml.spec.ts

```diff
@@ -172,3 +172,12 @@ describe("converts streetAddressLine to array correctly", () => {
     expect(result).toEqual(jsWithMultipleAddressLine)
   })
 })
+
+describe("readXml reads XML chars in attributes", () => {
```

Comment:
the escaped chars here are in the node text not the attribute. The contents of the test look fine though, just the name is wrong.

Reply:
Thanks Matt. Have edited that test name now.

So in this PR `medication` and `patientInfo` can handle the XML special chars.

Should `controlledDrugWords` and `additionalInstructions` also be made to handle these, or are these fields not ever expected to contain those chars?